SU-SOE/stanford_sites_jumpstart_engineering
# SOE-1695: Enable gallery_block and gallery_administration

# READY FOR REVIEW!!

# Summary
Would you look at that? We never enabled these modules in JSE!

# Needed By (Date)
- Thursday, 3/16

# Urgency
- Not urgent

# Steps to Test

1. Build JSE production 
2. Run behat tests:` $../../bin/behat -p your-jse-prod -s dev features/stanford_gallery`
3. Verify they pass!

# Affected Projects or Products
- JSE Product

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1695
- Other PRs: https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/133, and https://github.com/SU-SWS/linky_clicky/pull/149


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
